build.gradle

```diff
@@ -261,52 +269,35 @@ def getGitHash = { ->
 
 
 // -- Test tasks --
-// The default 'test' task runs all of the tests.
+// The default 'test' task runs all of the unit tests.
 // There are separate tasks for running just the unit tests and just the connected tests.
 // connected tests and running all of the tests.
-//
-// Configure test logging to stun - these settings mean that we get sufficient debug data when tests fail
-// during travis run.
-// TODO: I looked for a simple way to avoid redundancy in these tasks and didn't find it.
 test {
     useJUnit {
         includeCategories 'bio.terra.category.Unit', 'bio.terra.category.StairwayUnit'
-        excludeCategories 'bio.terra.category.Connected', 'bio.terra.category.Integration'
-    }
-    testLogging {
-        outputs.upToDateWhen {false}
-        showStandardStreams = true
-        showStackTraces = true
-        showCauses = true
-        exceptionFormat = "full"
     }
 }
 
 task testConnected(type: Test) {
     useJUnit {
         includeCategories 'bio.terra.category.Connected'
-        excludeCategories 'bio.terra.category.Unit', 'bio.terra.category.StairwayUnit', 'bio.terra.category.Integration'
     }
     testLogging {
-        outputs.upToDateWhen {false}
+        events = ["passed", "failed", "skipped"]
         showStandardStreams = true
-        showStackTraces = true
-        showCauses = true
-        exceptionFormat = "full"
     }
 }
 
 task testIntegration(type: Test) {
     useJUnit {
         includeCategories 'bio.terra.category.Integration'
-        excludeCategories 'bio.terra.category.Unit', 'bio.terra.category.StairwayUnit', 'bio.terra.category.Connected'
     }
-    testLogging {
-        outputs.upToDateWhen {false}
-        showStandardStreams = true
-        showStackTraces = true
-        showCauses = true
-        exceptionFormat = "full"
+}
+
+task testUnit(type: Test) {
+    useJUnit {
+        includeCategories 'bio.terra.category.Unit'
+            'bio.terra.category.Integration'
```

Comment:
why are we testing unit and integration tests in this task?
also in testAll below, i'm wondering if it makes sense to lump the integration tests in as well since this needs different set up than the rest of the tests? (i think i added it - just wondering)

Reply:
I am unsure how this ended up this way -- good catch. Yes integration tests should be separate. I'll update this.